server/edp/server/experiment.py

```diff
@@ -33,15 +33,15 @@ def __init__(self):
     )
     def create(self, experiment):
         self.requireParams(['startDate', 'title', 'motivation', 'experimentalDesign',
-                            'experimentalNotes', 'dataNotes', 'completed'], experiment)
+                            'experimentalNotes', 'dataNotes'], experiment)
```

Comment:
@alesgenova It doesn't make a lot of sense to require this on creation. So I have simply defaulted it to `False`